views/docs_home.html

```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
     <!-- end of Getting Started -->
     <!-- Examples -->
     <!-- end of Examples -->
+    <!-- Tutorials -->
+    <div class="docs-section">
+      <h2>
+        Tutorials
+      </h2>
+      <p class="description">
+        Learn DataHub features by tutorials!
+      </p>
+      <div class="row">
+        <a href="http://datahub.io/examples/simple-graph-spec" class="section-link">
+          <div class="col-md-4">
+            <h4>Graphs</h4>
+            <p>Tutorial on how to build the simple and nice graphs using the <b>Simple Graph Spec</b>. </p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/geojson-tutorial" class="section-link">
+          <div class="col-md-4">
+            <h4>GeoJSON</h4>
+            <p>Tutorial on how to package up <b>GeoJSON</b> data and display it on the map.. </p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/vega-views-tutorial-lines" class="section-link">
+          <div class="col-md-4">
+            <h4>Visualizations</h4>
+            <p>Tutorial on how to build visualizations using the <b>Vega Graph Spec</b></p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/vega-views-tutorial-topojson" class="section-link">
+          <div class="col-md-4">
+            <h4>TopoJSON</h4>
+            <p>Tutorial on how to build visualizations using the <b>Vega Graph Spec</b></p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/vega-views-tutorial-grouping" class="section-link">
+          <div class="col-md-4">
+            <h4>Grouping</h4>
+            <p>Tutorial on how to build visualizations using the <b>Vega Graph Spec</b></p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/transform-example-gdp-uk" class="section-link">
+          <div class="col-md-4">
+            <h4>Transform</h4>
+            <p>Tutorial on how data transforms work</p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/example-sample-transform-on-currency-codes" class="section-link">
+          <div class="col-md-8">
+            <h4>Transform</h4>
+            <p>Tutorial on how data transforms works</p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/transform-examples-on-co2-fossil-global" class="section-link">
+          <div class="col-md-4">
+            <h4>Transform</h4>
+            <p>Tutorial on how data transforms works</p>
```

Comment:
In this example, we explain how filtering and applying formula can be done before dataset gets rendered in showcase page.